javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
Yes, it's certainly possible and handled within the codebase (e.g. https://github.com/blakeembrey/javascript-stringify/blob/ecba78dca1c567a291e353ba0c6cbe6f41965e14/javascript-stringify.js#L127 and returned from https://github.com/blakeembrey/javascript-stringify/blob/ecba78dca1c567a291e353ba0c6cbe6f41965e14/javascript-stringify.js#L241). I'd love to improve the docs on this, definitely shouldn't let you get bitten by it. `undefined` comes back when the value can't be stringified because it's unknown.